cpp/llm_chat.cc

```diff
@@ -707,6 +707,44 @@ class LLMChat {
     }
   }
 
+  std::string RawGenerate(std::string prompt, int64_t generate_len) {
```

Comment:
Just curious (not asking for a change), is there a way to control the number of tokens in the prompt?

Reply:
We can iterate